lcls_tools/common/devices/yaml/GUNB.yaml

```diff
@@ -1,3 +1,28 @@
+lblms:
+  LBLM00A:
+    controls_information:
+      PVs:
+        bypass: LBLM:GUNB:212:A:FAST_AMP_BYP
+        gain: LBLM:GUNB:212:A:FAST_AMP_GAIN
+        i0_loss: LBLM:GUNB:212:A:I0_LOSS
+      control_name: LBLM:GUNB:212:A
+    metadata:
+      area: GUNB
+      beam_path: []
```

Comment:
Why no beam paths here?

Reply:
lcls_elements doesn't have any beam paths, for any LBLM devices unfortunately.